packages/react-composites/src/composites/common/CaptionsBanner.tsx

```diff
@@ -11,7 +14,7 @@ import { useAdaptedSelector } from '../CallComposite/hooks/useAdaptedSelector';
 import { useHandlers } from '../CallComposite/hooks/useHandlers';
 /* @conditional-compile-remove(close-captions) */
 import { _captionsBannerSelector } from '@internal/calling-component-bindings';
-
+/* @conditional-compile-remove(close-captions) */
```

Comment:
Do we need to CC the whole export? I guess we don't need this line, 
Then we don't need these lines as well

```
// This is a placeholder to bypass CC of "close-captions", remove when move the feature to stable
export {};
```